src/Components/Home/Styles/bottom.css

```diff
@@ -0,0 +1,229 @@
+*{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+  }
+
+  
+  .head-container{
+    position:relative; 
+    width: 100%;
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+  .head-container img{
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+    filter: brightness(0.3);
+    z-index: 0;
+  }
+  
+  
+  .bg-text-overlay{
+    position: absolute;
+    margin-top: 7vh;
+    height: 30rem;
+    width: 80%;
+    display: flex;
+    color: white;
+  }
+  .bg-info.game1{
+    
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+  }
+  .bg-info-header{
+    font-family: 'Courier New', Courier, monospace;
+    padding: 1rem 1rem 0rem 1rem;
+    font-size: 2rem;
+  }
+  .bg-info-content{
+    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+    padding: 0rem 1rem 1rem 1rem;
+    font-size: 2rem;
+  }
+  .bg-info-price{
+    display: flex;
+    flex-direction: row;
+    padding: 1rem;
+    font-size: 2rem;
+    
+    gap: 1rem;
+    align-items: center;
+  }
+  .bg-info-price button{
+    all: unset;
+    font-size: 1.2rem;
+    background-color: #CD3D0F;
```

Comment:
Just a suggestion, use variables for colors

Reply:
Ha yes will work on that